### ISSUE TITLE
depencency in homebrew formula not work

### ISSUE THREAD

**Issue by sharkAndshark** [urbica/martin#925]
```shell
brew tap maplibre/martin https://github.com/maplibre/martin.git
...
brew install maplibre/martin/martin
...
martin
dyld[9686]: Library not loaded: /usr/local/opt/openssl@3/lib/libssl.3.dylib
  Referenced from: <DAE79476-D3D5-3F00-B489-E59C2519A80E> /opt/homebrew/Cellar/martin/0.8.7/bin/martin
  Reason: tried: '/usr/local/opt/openssl@3/lib/libssl.3.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/opt/openssl@3/lib/libssl.3.dylib' (no such file), '/usr/local/opt/openssl@3/lib/libssl.3.dylib' (no such file), '/usr/local/lib/libssl.3.dylib' (no such file), '/usr/lib/libssl.3.dylib' (no such file, not in dyld cache)
zsh: abort      martin
```

**Comment by nyurik** [comment]
note that openssl was required for v0.8.7, but should not be required for v0.9+

**Comment by nyurik** [comment]
closing as this has been fixed with 0.9+
